app/services/authorization/enterprise_service.rb

```diff
@@ -37,7 +37,7 @@ def initialize(token)
   # Return a true ServiceResult if the token contains this particular action.
   def call(action)
     allowed =
-      if token.nil? || token.token_object.nil? || token.expired?
+      if OpenProject::Configuration.ee_manager_visible? && (token.nil? || token.token_object.nil? || token.expired?)
```

Comment:
Don't we have two uses cases? You either want to find out whether a given feature is available or whether you want to show banners in case you don't have it.

That would mean that `Token.allows_to? ...` should return the same value regardless of the configuration value.